### ISSUE TITLE
Best way to run TRUST4 on TCR-sequencing data obtained from the Illumina MiSeq platform

### ISSUE THREAD

**Issue by dcarbajo** [liulab-dfci/TRUST4#285]
Hello again!

I want to process the sequencing data obtained by Croce _et al_ in **"Phage display profiling of CDR3β loops enables machine learning predictions of NY-ESO-1 specific TCRs"** with TRUST4.

They build large phage display libraries of TCRs with randomized CDR3 β chain and pan them against the NY-ESO-1 epitope. They performed 2x250bp paired-end NGS sequencing on the extracted and amplified output DNA using an in-house Illumina MiSeq platform.

The TCR-sequencing data obtained with MiSeq was processed using MiXCR v3.0.13 with standard parameters (_mixcr analyze amplicon --species hs starting-material dna --5-end v-primers --3-end j-primers --receptor-type TRB_).

Now I want to process this raw data myself using TRUST4 for other follow-up analysis we want to do. Is there any parameter or anything else in particular that I should be aware of when running TRUST4 with this MiSeq data?

In principle, I just intended to run it like this (per sample):

```
run-trust4 --barcodeLevel cell
           -f path_to/hg38_bcrtcr.fa
           --ref path_to/human_IMGT+C.fa
           -1 path_to/file1.fastq.gz
           -2 path_to/file1.fastq.gz
           --repseq -o sample_name --od sample_output_dir --clean 1 -t 8
```

Thanks a lot for all your help!

**Comment by mourisl** [comment]
Yes, it seems this is a non-UMI-based TCR-seq data, so using --repseq is good. There is no need to specify the barcodeLevel as barcode information is not provided. 

The starting material is DNA, so there might be many un-recombined genomic sequences in the output, which may require extra step for filtering, like mapping to the reference genome.

Hope this helps.
